pkg/framework/context/context.go

```diff
@@ -110,19 +116,28 @@ func (p *Provider) InboundTransportEndpoint() string {
 	return p.inboundTransportEndpoint
 }
 
+func (p *Provider) tryToHandle(svc service.InboundHandler, msg service.DIDCommMsgMap, myDID, theirDID string) error {
```

Comment:
is this method temporary until the inbound handler is substituted by the messenger?

Reply:
actually I did not think about it, I thought it is enough. @troyronda What do you think?

Reply:
see comment about loop.

Reply:
@soluchok I believe this function is *temporarily* handling the msg twice, right? The end-goal is to have only the messenger handle the msg, correct?

Reply:
It does not handle the message twice.

Reply:
Yes it is.... the message is being handled by the messenger and then by the service.